frontend/source/components/access.svelte

```diff
@@ -430,11 +452,15 @@
 				</div>
 			</div>
 			<div class="form-row d-flex justify-content-center mt-2">
-				<div bind:this={type_btn_group} class="btn-group btn-group-toggle flex-wrap" data-toggle="buttons">
+				<div bind:this={type_btn_group} class="btn_padded btn-group btn-group-toggle flex-wrap" data-toggle="buttons">
 					<label class="btn btn-secondary shadow-none"><input type="radio" name="options"/>posts</label>
 					<label class="btn btn-secondary shadow-none"><input type="radio" name="options"/>comments</label>
 					<label class="btn btn-secondary shadow-none active"><input type="radio" name="options"/>all</label>
 				</div>
+				<div bind:this={sort_btn_group} class="btn_padded btn-group btn-group-toggle flex-wrap" data-toggle="buttons">

```

Comment:
i see youve added a btn group. i think it would be a better design to simply have a small toggle btn with visual indicator (e.g., unpressed for desc, pressed for asc) for this sort feature, rather than a whole new btn group, to avoid bloating up the ui

ideal position for it
![image](https://user-images.githubusercontent.com/31039318/193328333-3995cb81-ab52-42bf-a4ed-b4c73aa6ae17.png)
(the btn should visually fit with that row. the size of the subreddit select should be decreased to make space for it, and there should be spacing between the subreddit select and the new btn, like the search bar and the subreddit select)

Reply:
Will try to implement this, might take a bit since I'm unfamiliar with svelte and JS